### DISCUSSION TITLE
can't get "Move scheduled message to this Thunderbird" to work

### DISCUSSION THREAD

**Discussion by G-DRF** [Extended-Thunder/send-later#673] [Q&A]
Wonder if anyone has success with moving scheduled messages between different instances of Thunderbird?  

I write lots of emails in the evening at home that I don't want to send till the next business day (so as not to bother people much after hours) so I type them and schedule to send later.  Of course by the next business day I'm at work on my computer there and my computer at home is off.

I tried "Move scheduled message to this Thunderbird" from the context menu but it does not seem to do anything, messages do not send, nor do they show up in the Send Later drop down menu.  

Also tried the old way (worked great in previous Thunderbird) of opening the draft message in my work instance of Thunderbird and doing a small edit like adding a space and then clicking send later, click to send at the time I scheduled the night before but the message doesn't send, just sits there in Drafts.  Tried changing the send later time by a few minutes and that didn't work either.

I checked and my identity is exactly the same in both my home and work Thunderbird profile settings.

Am I missing something? 

Thanks for any ideas!

**Comment by jikamens** [comment]
I am unable to reproduce the problem you're having. "Move scheduled message to this Thunderbird" works fine for me, and the moved message gets sent as expected.

I did notice an issue where sometimes Thunderbird on one computer failed to detect changes made to my Drafts folder on a different computer, but that's a TB issue, not a Send Later issue, and it's not the problem you described here.

Is the mailbox in question on Gmail, if so, that may be relevant, and there are some fixes in the release I'm about to put out that might make it work better.

Are you sure the issue is specifically moved / edited messages don't send, as opposed to Send Later not being able to send scheduled messages on your work computer _at all_? What happens when you create a new message on your work computer and schedule it for five minutes into the computer? Does it send after five minutes?
